### PR TITLE
[0.1.1] Add models

### PR DESCRIPTION
**Pull request by Voovaaa**

### CODE REVIEW

File: mini_messenger/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Generated by Django 4.1.2 on 2022-11-06 07:06
+
+from django.conf import settings
+import django.contrib.auth.models
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
```
Comment (Safintim) [inline mini_messenger/migrations/0001_initial.py:10]: add pycache to gitignore, db.sqlite3

File: mini_messenger/models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+class Dialog(models.Model):
+    users = models.ManyToManyField(to=CustomUser)
+
+
+class Message(models.Model):
+    text = models.TextField()
+    datetime_sent = models.DateTimeField(timezone.now)
+    sender = models.ForeignKey(to=CustomUser, on_delete=models.DO_NOTHING)
+    dialog = models.ForeignKey(to=Dialog, on_delete=models.CASCADE)
+#from mini_messenger.models import CustomUser, Dialog, Message
```
Comment (Safintim) [inline mini_messenger/models.py]: remove comments

File: mini_messenger/models.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    contacts = models.ManyToManyField(to='CustomUser')
+
+
+class Dialog(models.Model):
+    users = models.ManyToManyField(to=CustomUser)
+
+
+class Message(models.Model):
+    text = models.TextField()
+    datetime_sent = models.DateTimeField(timezone.now)
```
Comment (Safintim) [inline mini_messenger/models.py]: created_at

Comment (Safintim) [reply mini_messenger/models.py]: models.DateTimeField(timezone.now) what is mean? is auto_now or auto_now_add. Need more concrete

File: mini_messenger/models.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    sender = models.ForeignKey(to=CustomUser, on_delete=models.DO_NOTHING)
```
Comment (Safintim) [inline mini_messenger/models.py]: why not cascade?

File: mini_messenger/migrations/0001_initial.py (hunk 1 above)
Comment (Safintim) [reply mini_messenger/migrations/0001_initial.py:10]: Add .idea to gitignore and remove from repo

File: mini_messenger/models.py (hunk 4 above)
Comment (Voovaaa) [reply mini_messenger/models.py]: because it will show "Deleted user object" in future

File: mini_messenger/migrations/0001_initial.py [hunk 5]
```diff
@@ -0,0 +1,89 @@
+# Generated by Django 4.1.2 on 2022-11-09 12:24
+
```
Comment (Voovaaa) [inline mini_messenger/migrations/0001_initial.py:2]: remove db.sqlite3

Comment (Voovaaa) [reply mini_messenger/migrations/0001_initial.py:2]: remove pycache

remove idea

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,4 @@
-mini_messenger/.idea
-mini_messenger/db.sqlite3
+.idea
+db.sqlite3
+mini_messenger/__pycache__
```
Comment (Safintim) [inline .gitignore]: __pycache__

File: .gitignore [hunk 7]
```diff
@@ -1,2 +1,4 @@
...
+mini_messenger/migrations/__pycache__
```
Comment (Safintim) [inline .gitignore]: remove

**Review by Safintim — APPROVED** [review]

**Pull request merged**